From 0b46212d99191b65a1dd2f723c9cf0250e69a448 Mon Sep 17 00:00:00 2001
From: Nils Philippsen <nils@redhat.com>
Date: Wed, 2 Feb 2011 18:19:35 +0100
Subject: [PATCH] patch: pyslice

Squashed commit of the following:

commit 0b91c365bc1d43a888ed9b78439930c1235b609c
Author: Nils Philippsen <nils@redhat.com>
Date:   Wed Feb 2 17:11:28 2011 +0100

    Bug 641259 - [abrt] gimp-2:2.6.11-1.fc14: py-slice.py:172:slice:TypeError: integer argument expected, got float

    py-slice: cast cellspacing to int in pyslice() to avoid tracebacks
    (cherry picked from commit 0af966b63fcc55b36380d6538dfb30000f71fef9)
---
 plug-ins/pygimp/plug-ins/py-slice.py |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/plug-ins/pygimp/plug-ins/py-slice.py b/plug-ins/pygimp/plug-ins/py-slice.py
index 40743f3..ac35f23 100755
--- a/plug-ins/pygimp/plug-ins/py-slice.py
+++ b/plug-ins/pygimp/plug-ins/py-slice.py
@@ -36,6 +36,9 @@ gettext.install("gimp20-python", gimp.locale_directory, unicode=True)
 def pyslice(image, drawable, save_path, html_filename,
             image_basename, image_extension, separate,
             image_path, cellspacing, animate, skip_caps):
+
+    cellspacing = int (cellspacing)
+
     if animate:
         count = 0
         drw = []
-- 
1.7.4

